Assignment/solution_student/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,49 @@
 // 1.	Slide the header up when it is clicked on.
-
+$('header').on('click', function(){
+	$('header').slideUp('slow');
+});
 // 2.	Remove the purplebox class
+$('*').removeClass('purplebox');
 
 // 3.	Hide all of the div’s in #row4
+$('#row4').hide();
 
 // 4.	Add a paragraph that says “I have been added” to all boxes with class “add-para” (Hint: you'll need to look up the append() method)
+$('.add-para').append('I have been added');
 
 // 5.	Add a class in your css and add it to all items with the class .box
 
+$('.box').addClass('fun');
+
 // 6.	Show the footer when the anchor in #box2 is clicked
 
+$('#box2').on('click', function(){
+	$('footer').show('slow');
+});
+
 // 7.	Change the styling of the span inside of #box3 by adding a class (and style that class in CSS)
+$('p>span').addClass('spanFun');
```

Comment:
This selection is still kind of generic, because it could be any p with a span, but it wants you to target the span inside of #box3 specifically, so I would alter my selection to `$('#box3 span')`